src/main/java/org/osc/sdk/manager/element/ManagerSecurityGroupInterfaceElement.java

```diff
@@ -34,10 +36,21 @@
      */
     String getName();
 
-    /**
-     * @return the identifier of this policy defined by the security manager
-     */
-    String getPolicyId();
+	/**
+	 * Provide manager security group id if the manager is supporting policy
+	 * mapping and security group.
+	 *
+	 * @return the manager security group identifier
+	 */
+	String getManagerSecurityGroupId();
+
+	/**
+	 * Provide manager policy id in case of IP/MAC or encapsulation tag in case
+	 * security appliance supports encapsulation.
+	 *
+	 * @return the set of policy id's
```

Comment:
id ->  identifiers (also check other places). Also maybe:
the set of policy identifiers or encapsulation tags supported by this manager